src/testsuite/vec_perf_f32.c

```diff
@@ -54,8 +53,10 @@ extern __vector bool int
 test_pred_f32_subnormal (vf32_t value);
 extern __vector bool int
 test_pred_f32_zero (vf32_t value);
+#ifndef __clang__
```

Comment:
why? add a comment.

Reply:
// Clang does not define vec_test_data_class and friends.